README.md

```diff
@@ -73,55 +73,44 @@ Then build the docker container and push it to Google Cloud Run:
 
 ## Training the model
 
-If you have a GPU installed on your system, then you can do training.
+__WARNING__: Training takes substantial computing resources. Some datasets are large and require significant computing power to preprocess and significant RAM as working space (although preprocessing does not load all data in memory for embedding vector datasets). At least one NVIDIA GPU is also required.
 
 Run the following commands from the root directory of the repository.
 
-1. Install development requirements.
+1. Load the environment and install development requirements.
 ```
+source .env
 pip install -r dev-requirements.txt
 ```
 
-2. Download and prepare the dataset. This requires over a GB of disk space.
-```
-./build/prepare-wmt14 --small
-```
-To prepare the full dataset, omit the `--small`. Preparation for the full dataset can take 15GB of disk space and 10-15 minutes on a fast computer. 
-
-3. Activate the virtual environment if it is not already activated.
+2. Download and prepare a small dataset of a few hundred MB. For more data, a larger data source or multiple data sources can be used. Possible data sources are listed using `python run.py pepare-data list-datasets`.
 ```
-source fast-training/bin/activate
-source .env
+python run.py prepare-data bert fr bert_fr_en "['news2014']" 50 --shuffle True --valid-size 4096
```

Comment:
Error: ModuleNotFoundError: No module named 'dask'

After I pip install dask, I get this error:

Traceback (most recent call last):
  File "run.py", line 17, in <module>
    from corpus import (
  File "/mnt/fs03/home/nlopatina/fast-training/src/corpus.py", line 8, in <module>
    from pytorch_pretrained_bert import BertTokenizer, BertModel
ModuleNotFoundError: No module named 'pytorch_pretrained_bert'